src/app/components/profile-page/profile-page.component.html

```diff
@@ -1,8 +1,20 @@
 <main class="container px-1">
-  <div *ngFor="let image of images" class="img-container m-1">
-    <a [routerLink]="['image']" [queryParams]="{imageName: image.imageName, imageUrl: image.imageUrl }"
-      queryParamsHandling="merge">
-      <img class="item-img w-100 h-100" [src]="image.imageUrl" alt="Picture">
-    </a>
+  <ng-container *ngIf="!eventService.deleteMode else removableItems">
+    <div *ngFor="let image of images" class="img-container m-1">
+      <a [routerLink]="['image']" [queryParams]="{imageName: image.imageName, imageUrl: image.imageUrl }"
+        queryParamsHandling="merge">
+        <img class="item-img w-100 h-100" [src]="image.imageUrl" alt="Picture">
+      </a>
+    </div>
+  </ng-container>
+</main>
+<ng-template #removableItems>
```

Comment:
can this be inside of the <main> element?

Reply:
Yes it can.

Reply:
Done